config/job.yaml.tim

```diff
@@ -39,9 +44,25 @@ spec:
         volumeMounts:
         - mountPath: /opt/screwdriver
           name: screwdriver
+        - mountPath: /var/run/sd
+          name: logs
         env:
         - name: SD_TOKEN
           value: "{{token}}"
+      - name: logservice
+        image: golang:1.7
```

Comment:
To be more specific, we need this repo https://github.com/screwdriver-cd/log-service to have a Dockerfile in it. The image listed in this job definition should point to the one generated by the log-service repo.